homeassistant/components/mqtt/cover.py

```diff
@@ -401,11 +402,12 @@ def supported_features(self):
 
         return supported_features
 
-    async def async_open_cover(self, **kwargs):
+    def open_cover(self, **kwargs):
```

Comment:
Don't change this, we want to keep using the async friendly version.

Reply:
Done! Still thinking in Java, where i would implement the sync methods to do the work and the async methods would only start the sync methods in threads (workers)